### ISSUE TITLE
output logits from CNN

### ISSUE THREAD

**Issue by xukai92** [xukai92/interpreting-deep-classifier#3]

**Comment by xukai92** [comment]
it's fine that CNN output weights. only need to convert them to logits when creating the oracle data
